specs/latest/2.0/index.html

```diff
@@ -3292,14 +3292,20 @@ <h3><a name="BUFFER_OBJECT_BINDING">Buffer Object Binding</a></h3>
         from other buffer data also maps better to the Direct3D API.
     </div>
 
+    <h3>Preventing undefined behavior with Transform Feedback</h3>
+
     <p>
-        In addition, a buffer can not be simultaneously bound to a transform feedback buffer binding point
-        and another buffer binding point in the WebGL 2.0 API. Attempting to violate this rule generates an
-        <code>INVALID_OPERATION</code> error, and the state of the binding point will remain untouched.
+        Attempting to read from or write to a buffer which is bound to a transform feedback buffer binding
+        point, via any mechanism other than transform feedback itself (e.g. as a vertex array in a draw
+        call), fails with an INVALID OPERATION error. There is one exception to this rule: bufferData,
+        bufferSubData, getBufferSubData, and copyBufferSubData are allowed as long as transform feedback
+        is not active.
     </p>
 
     <div class="note rationale">
-        This restriction has been added to prevent undefined values when reading from or writing to a buffer object that is simultaneously bound to a transform feedback buffer binding point and elsewhere in the GL. An example of such a situation is <code>ReadPixels</code> to a pixel buffer object binding point. See <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-2.15.2">OpenGL ES 3.0.4 &sect;2.15.2</a>)</span>.
+        If an error were not generated, the values read or written would be undefined. <span class="gl-spec">
```

Comment:
Not necessary, but: could be nice to keep some of the rationale that was there before (like the example) (updated to the new semantics).